src/pages/MapViewer/components/Toolbox/components/ToolboxCollections/index.tsx

```diff
@@ -7,15 +7,15 @@ import ToolboxRow from '../ToolboxRow';
 import './styles.scss';
 
 const ToolboxCollections: React.FC = () => {
-  const { state: MapSettingsState } = useMapSettings();
-  const { toolboxCollectionsResults: toolboxCollectionsResults } = MapSettingsState;
+  const { collections } = useMap();
```

Comment:
A reason why I named it `toolboxCollectionsResults` is because the collections visible here will be different to the collections visible on the `DataCatalogue`

Reply:
Fair point but it is in a section of code called toolbox, so to me it is obvious that it relates to toolbox collections. If we start naming everything with it's area of concern, then it could get very long, why not use `mapToolboxCollectionsResults` for instance, then we know it is to do with the map toolbox? I'm not against the naming so if you think it is really important I will, but I don't see it adds any information myself.

I don't think there is a right/wrong answer on this, they are all collections at the end of the day aren't they? 

What is the difference between a collection here and a DataCatalogue collection? Even if there is a difference, they should have different types and that will help us distinguish, if necessary.

Reply:
I don't mind changing it back but we are in a component that already has `toolbox` in it, would we want to rename every variable to include toolbox? I just see it as extra typing myself and adds nothing to my understanding of the code. However if you have a strong opinion on this, I'm happy to change it.

Reply:
It largely depends on how much cross over is planned between the map and the `DataCatalogue`. The only time this could potentially raise a problem would be if we are sharing collection state within a component. 

Leave for now, we can go through and add naming separation if we get a ticket that leads to this sharing of state.